### DISCUSSION TITLE
how to turn off https login page

### DISCUSSION THREAD

**Discussion by NDWCZ** [AdguardTeam/AdGuardHome#4445] [Q&A]
someone trying to login the adguard server by https without dns-query,How to turn off HTTPS automatically jump to the login page

**Comment by ainar-g** [comment]
It is not currently possible to do that with AGH only, see #1964.  You can use your OS's firewall in the meantime.

**Comment by NDWCZ** [comment]


**Reply by cxw620** [reply]
> 您好，您的邮件我已收到，紧急情况请致电：13078033735。谢谢

兄弟, 电话号码隐藏一下吧
